src/app/view/components/dot-relationship-tree/dot-relationship-tree.component.html

```diff
@@ -0,0 +1,30 @@
+<ul class="dot-tree">
+    <li>
+        <div class="dot-tree__content">
+            <dot-icon-button icon="wysiwyg"></dot-icon-button>
+            <span>{{ parent }}</span>
+        </div>
+        <ul class="dot-tree--child">
+            <li>
+                <div class="dot-tree__content">
+                    <dot-icon-button
+                        icon="wysiwyg"
+                        [ngClass]="'dot-tree--active'"
```

Comment:
`[ngClass]` is when you need to add the class dynamically. If not, you can use just `class.`